gslab_scons/tests/test_log.py

```diff
@@ -1,39 +1,187 @@
-#! /usr/bin/env python
-
 import unittest
 import sys
 import os
+import mock
+import time
+# Import gslab_scons testing helpers
+import _test_helpers as helpers
 
 # Ensure that Python can find and load the GSLab libraries
 os.chdir(os.path.dirname(os.path.realpath(__file__)))
 sys.path.append('../..')
 
-from gslab_scons import log_timestamp
+import gslab_scons.log as gs
+import gslab_scons.misc as misc
+import gslab_scons._exception_classes as ex_classes
+from gslab_make.tests import nostderrout
+
+# Define path to the builder for use in patching
+path = 'gslab_scons.log.misc'
+
+
+class TestLog(unittest.TestCase):
+
+    # I think that we should mock check_lfs in all tests
+    # so that the tests run irrespectively of whether the
+    # user has git_lfs
+
+    def setUp(self):
+        if os.path.isfile('sconstruct.log'):
+            os.remove('sconstruct.log')
+
+    # Mock a Unix platform (sys.platform = 'darwin' on  Mac machines).
+    @helpers.platform_patch('darwin', path)
+    @mock.patch('gslab_scons.log.misc.check_lfs')
+    def test_start_log_stdout_on_unix(self, mock_lfs):
```

Comment:
As discussed, adding a `@mock.patch()` decorator without a second argument above a method passes that method a MagicMock of the object specified by the patch's first argument.  [This documentation](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) may be useful.